### PR TITLE
libvirt-howto: Less intrusive NetworkManager setup

### PR DESCRIPTION
**Pull request by zeenix**
Instead of modifying the main NetworkManager configuration, let's add a separate configuration file for our purposes.

Based on a patch from Colin Walters: https://github.com/code-ready/osp4/pull/36

### CODE REVIEW

File: docs/dev/libvirt-howto.md [hunk 1]
```diff
@@ -192,7 +192,12 @@ sudo virsh pool-autostart default
 ### Set up NetworkManager DNS overlay
 
 This step allows installer and users to resolve cluster-internal hostnames from your host.
-1. Edit `/etc/NetworkManager/NetworkManager.conf` and set `dns=dnsmasq` in section `[main]`
+1. Tell NetworkManager to use `dnsmasq`:
+
+```sh
```
Comment (wking) [inline docs/dev/libvirt-howto.md]: nit: this is part of the list entry, so the whole `sh` block should be indented four spaces.

Comment (zeenix) [reply docs/dev/libvirt-howto.md]: fixed

**Pull request merged**